docs/app/components/input/input-demo.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="demo-content">
+    <h1>Input Text Field</h1>
+    <h6>Last updated {{lastModified | date:'longDate'}}</h6>
+
+    <hc-tab-set direction="horizontal">
+    <hc-tab tabTitle="Overview">
+
+    <hc-tile>
+        <h5>About</h5>
+        <p>Text fields allow users to input, edit, and select text.
+            They typically reside inside forms, but can also be used in modal dialogs and subnavigation bars.  Input text fields are also used for
+            search and filtering of content.</p>
+    </hc-tile>
+
+    <hc-tile>
+        <h5>Example</h5>
+        <h4>Input with Validation</h4>
+        <form (ngSubmit)="onSubmit()">
+            <hc-input [valid]="validCheck" required="true">
```

Comment:
Since `hc-input` is tied into angular forms, we probably don't need to add our own valid input as it's given to us for free. https://angular.io/guide/form-validation

Reply:
Good suggestion Cory - just recapping what we discussed on slack here.  Instead of hc-input being a separate component, I'll adjust so that it actually extends the base input component to `input [hc-input]` - to take advantage of all the built in form validation.  Then in addition I'll create `label [hc-label]` - which is better because it can be used by both select and input that way.  And probably keep hcErrors as a directive that can be used by any form component.  I'll also take a look and see if it would be possible to be consistent with our select component and switch it to `select [hc-select]`.